src/sentry/snuba/outcomes.py

```diff
@@ -0,0 +1,29 @@
+from sentry.utils.snuba import (
+    raw_query,
+)
+from sentry.snuba.dataset import Dataset
+from sentry_relay import DataCategory
+
+
+def query(groupby, start, end, rollup, aggregations, orderby, fields=None, filter_keys=None):
+
+    # TODO: if roll up >= use hourly, else use raw
+    # add spans
+    result = raw_query(
+        start=start,
+        end=end,
+        groupby=groupby,
+        aggregations=aggregations,
+        rollup=rollup,
+        filter_keys=filter_keys,
+        orderby=orderby,
+        dataset=Dataset.Outcomes,
+    )
+
+    for row in result["data"]:
+        row["category"] = (
```

Comment:
This is likely simpler than trying to cook up a case statement that will yield a similar result in clickhouse.